_posts/2023-04-12-feb-2023-roundup.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "What we've been reading in February"
+author: tyler
+tags: [roundup]
+---
+
+<!-- excerpt start -->
+
+Here are the articles, videos, and tools that we've been excited about this
+February. We're a little late with the February issue but we'll catch up!
+
+<!-- excerpt end -->
+
+We hope you enjoy these links, and we look forward to hearing what you've been
+reading in the comments or [on the Interrupt Slack](https://interrupt-slack.herokuapp.com/).
+
+## Articles & Learning
+
+- [**The cheapest flash microcontroller you can buy is actually an Arm Cortex-M0+ - Jay Carlson**](https://jaycarlson.net/2023/02/04/the-cheapest-flash-microcontroller-you-can-buy-is-actually-an-arm-cortex-m0/)<br>
+Jay Carlson is at it again with his latest post, showing that a Cortex-M0+ might be the best chip when you need to be cheap. 
+
+- [**ESP32 Buyer’s Guide: Different Chips, Firmware, Sensors - eitherway**](https://eitherway.io/posts/esp32-buyers-guide/)<br>
+A comparison of all of the ESP32 (and ESP8266) flavors that one can buy. 
+
+- [**Rust on Espressif chips - 2023 Roadmap**](https://mabez.dev/blog/posts/esp-rust-24-02-2023/)<br>
+Rust on ESP32 is continuing to chug along. This post covers a 2022 Q4 recap and 2023 roadmap for Rust on the ESP32 platform.
+
+- [**RFID Reader for Apartment Door**](http://dlaw.me/door-rfid-reader/)<br>
+A beautifully documented project implementing a retrofitted RFID door lock, including mechanical + electrical + Rust firmware. Very polished final result! - Noah.
+
+- [**Improving the Beginner’s PID (Arduino)**](http://brettbeauregard.com/blog/2011/04/improving-the-beginners-pid-introduction/)<br>
+A nice article about done work that went into the Arduino PID library - Noah.
```

Comment:
```suggestion
A nice article about some work that went into the Arduino PID library - Noah.
```

Reply:
Hah, I fixed this but undid it to test ChatGPT. But never went back to fix it. Thank you!